lib/JsonpChunkTemplatePlugin.js

```diff
@@ -11,7 +11,15 @@ class JsonpChunkTemplatePlugin {
 		chunkTemplate.plugin("render", function(modules, chunk) {
 			const jsonpFunction = this.outputOptions.jsonpFunction;
 			const source = new ConcatSource();
-			source.add(`${jsonpFunction}(${JSON.stringify(chunk.ids)},`);
+			source.add("(function() {");
+			source.add(this.indent([
+				"if (typeof global !== 'undefined') { return global; }",
+				"if (typeof window !== 'undefined') { return window; }",
+				"if (typeof self !== 'undefined') { return self; }",
+				"throw new Error('unable to locate global object');"
```

Comment:
`this` and `global` in `Weixin` are different, but both could be as global object.